portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* nav buttons */
+
+var sections = ['.home', '.projects', '.cats', '.skills', '.contacts']
+
 /**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+ * Return function, which moves to the section with index 'sectionIndex' and,
+ * if the navbar ('nav') will hide part of the
+ * section's contect, scrolls up by the navbar height.
+ * @param  {int} sectionIndex [index number of the section]
```

Comment:
you don't need the square brackets (`[]`) here. So ` @param {int} sectionIndex Index number of the section.`

Same thing for return and for the other function